### DISCUSSION TITLE
How can I set DaxStudio as an add-in for Excel?

### DISCUSSION THREAD

**Discussion by gurpal-rattu** [DaxStudio/DaxStudio#1248] [Q&A]
Hello, my company only allows some software add-ins for office 365. I've tried searching for DaxStudio in the Microsoft store and I can't find it there, can someone tell me how to upload this app as a manifest?

**Comment by dgosbell** [comment]
The DAX Studio Excel addin component is what is known as a COM addin. It is not possible to publish this type of addin via the Microsoft Store. 

However, the Excel addin is only required if you need to connect to PowerPivot models in Excel. If you are using any other type of model (eg. Power BI, Analysis Services) then the Excel addin is not required.

**Reply by gurpal-rattu** [reply]
Hello,
I realized I made a mistake in my original request. This is actually for PowerPoint and not for Excel. How can I add this add-in for PowerPoint?

**Reply by dgosbell** [reply]
You can't - we don't have a PowerPoint add-in. How would you see this working? 

In Excel we can read the embedded Power Pivot model and output query results to the worksheets. But PowerPoint cannot host Power Pivot data models. I'm not sure what functionality you would want in PowerPoint, this request has never come up before.
